client/my-sites/email/email-providers-comparison/stacked/index.tsx

```diff
@@ -269,6 +312,35 @@ const EmailProvidersStackedComparison = ( {
 			{ ! isDomainInCart && domain && <EmailExistingPaidServiceNotice domain={ domain } /> }
 
 			<>
+				{ ! currentUserCanAddEmail && (
+					<PromoCard className="email-providers-stacked-comparison__non-owner-notice">
+						<p>
+							{ translate(
+								'An email solution can only be purchased by the domain owner. ' +
+									'To make a purchase, please {{link}}log in{{/link}} with the account that purchased the domain. ' +
+									"If you don't have access to that account, please reach out to the domain owner {{reachOutLink}}%(ownerUserName)s{{/reachOutLink}}",
+								{
+									components: {
+										link: <a href={ loginUrl } onClick={ () => onClickLink( 'user_login' ) } />,
+										reachOutLink: isPrivacyAvailable ? (
+											<a
+												href={ contactOwnerUrl }
+												onClick={ () => onClickLink( 'owner_contact' ) }
+												rel="noopener noreferrer"
+												target="_blank"
+											/>
+										) : (
+											<></>
+										),
+									},
+									args: {
+										ownerUserName,
+									},
+								}
+							) }
+						</p>
+					</PromoCard>
+				) }
```

Comment:
Something to consider: given how long the `render()` method already is, we could extract that block into a standalone component (in the same file but defined above `EmailProvidersStackedComparison`).

Reply:
As this message is going to be used for the next two tasks, the next one is doing that refactor and new component :)

https://github.com/Automattic/wp-calypso/pull/68158#issuecomment-1257827364